configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
Amber(Tools) as a distribution builds a lot of dependencies (of relevance to cpptraj: blas, lapack, arpack, netcdf, etc) that it doesn't depend on the system for, but readline here is slightly different  in that a copy is embedded inside of the cpptraj source tree, unlike those others.

I checked Debian, and there readline is not a dependency of bash. On Arch, it is though. Either way, a lot of the system depends on it though, from python to whatever.